tests/manage/pv_services/test_bz1729853.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf8 -*-
+
+import logging
+import textwrap
+import yaml
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier3
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.ocs.resources.objectconfigfile import ObjectConfFile
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_pvc_dict(name):
+    """
+    Creates new copy of PVC dict to tweak and use.
+    """
+    template = textwrap.dedent(f"""
+        kind: PersistentVolumeClaim
+        apiVersion: v1
+        metadata:
+          name: {name}
+        spec:
+          storageClassName: ocs-storagecluster-ceph-rbd
+          accessModes:
+           - ReadWriteOnce
+          resources:
+            requests:
+              storage: '2Gi'
+        """)
+    pvc_dict = yaml.safe_load(template)
+    return pvc_dict
+
+
+def get_deploymentconfig_dict(name, pvc_name):
+    template = textwrap.dedent(f"""
+        kind: 'DeploymentConfig'
+        apiVersion: 'v1'
+        metadata:
+          name: {name}
+        spec:
+          template:
+            metadata:
+              labels:
+                name: {name}
+            spec:
+              restartPolicy: 'Always'
+              volumes:
+              - name: 'cirros-vol'
+                persistentVolumeClaim:
+                  claimName: {pvc_name}
+              containers:
+              - name: 'cirros'
+                image: 'cirros'
+                imagePullPolicy: 'IfNotPresent'
+                volumeMounts:
+                - mountPath: '/mnt'
+                  name: 'cirros-vol'
+                command: ['sh']
+                args:
+                - '-ec'
+                - 'while true; do
+                       (mount | grep /mnt) && (head -c 1048576 < /dev/urandom > /mnt/random-data.log) || exit 1;
+                       sleep 20 ;
+                   done'
+                livenessProbe:
+                  exec:
+                    command:
+                    - 'sh'
+                    - '-ec'
+                    - 'mount | grep /mnt && head -c 1024 < /dev/urandom >> /mnt/random-data.log'
+                  initialDelaySeconds: 3
+                  periodSeconds: 3
+          replicas: 1
+          triggers:
+            - type: 'ConfigChange'
+          paused: false
+          revisionHistoryLimit: 2
+        """)
+    dc_dict = yaml.safe_load(template)
+    return dc_dict
+
+
+@tier3
+@pytest.mark.polarion_id("OCS-278")
```

Comment:
we already have a PR for this TC - https://github.com/red-hat-storage/ocs-ci/pull/999

Reply:
I was not aware of this, will drop the marker.